Description: Install 'sddm.service' in 'graphical.target'.
Author: Elias Probst
Origin: upstream, https://github.com/eliasp/sddm/commit/39d1f816873995dbf46d80c20709827e8ea9806a https://github.com/sddm/sddm/pull/340
Reviewed-by: Jonathan Riddell
Last-Update: 2015-03-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: sddm-0.11.0/services/sddm.service.in
===================================================================
--- sddm-0.11.0.orig/services/sddm.service.in
+++ sddm-0.11.0/services/sddm.service.in
@@ -11,4 +11,4 @@ Restart=always
 
 [Install]
 Alias=display-manager.service
-
+WantedBy=graphical.target
